include/fmt/color.h

```diff
@@ -578,7 +578,7 @@ inline std::basic_string<Char> vformat(
 */
 template <typename S, typename... Args, typename Char = char_t<S>>
 inline std::basic_string<Char> format(const text_style& ts, const S& format_str,
-                                      const Args&... args) {
+                                      Args&&... args) {
```

Comment:
This looks unrelated, please revert.

Reply:
Done, though this was done since I noticed it would bring the function more into alignment with the other examples of and documentation for functions using `make_args_checked`